src/bitcoind-res.rc

```diff
@@ -17,13 +17,13 @@ BEGIN
         BLOCK "040904E4" // U.S. English - multilingual (hex)
         BEGIN
             VALUE "CompanyName",        "Bitcoin"
-            VALUE "FileDescription",    "Bitcoind (OSS daemon/client for Bitcoin)"
+            VALUE "FileDescription",    "bitcoind (Bitcoin client as a JSON-RPC server)"
```

Comment:
But a Bitcoin node is useful even if you don't use its RPC functionality,
for example by connecting a lightweight client using P2P to it.

Bitcoind is bitcoin-qt without the GUI part, so it's a headless version -
regardless of how you use it.